packages/EasyErrorHandler/src/Processors/ErrorCodesGroupProcessor.php

```diff
@@ -60,10 +64,13 @@ public function process(): ErrorCodesDto
     /**
      * @param mixed[] $errorCodeNames
      */
-    private function determineCategoryName(array $errorCodeNames): string
+    private function determineCategoryName(array $errorCodeNames, bool $isEnum): string
     {
         $explodedErrorCodeNames = \array_map(
-            static function ($errorCodeName): array {
+            static function ($errorCodeName) use ($isEnum): array {
+                if ($isEnum === true) {
+                    $errorCodeName = \preg_replace('/([a-z])([A-Z])/u', '$1_$2', $errorCodeName);
```

Comment:
Let's move this variable inside `determineCategoryName`

Reply:
Done

Reply:
Thanks